Calculator/Services/NumberService.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Calculator.Services
+{
+    public class NumberService : INumberService
+    {
+        private string[] DefaultDelimiters = { ",", "\n" };
+        private string[] DelimiterSurroundingStrings = { "//", "[", "][", "]" };
+
+        public List<int> ParseNumbers(string numbers)
```

Comment:
There is a lot going on in this method, including a bunch of different responsibilities (yes, the single responsibility principle only mentions classes but it can fall to methods too). See what you can do about it.